app/services/charges/override_service.rb

```diff
@@ -35,6 +35,19 @@ def call
           filters_result.raise_if_error!
         end
 
+        if charge.applied_pricing_unit
+          conversion_rate = params.dig(:applied_pricing_unit, :conversion_rate).presence
```

Comment:
So if it's present in the request but not on the charge, we do not take the parameter into consideration ?